tasks/openhpc-repos.yml

```diff
@@ -0,0 +1,39 @@
+---
+# This task list can configure Slurm nodes to use older OpenHPC update
+# repositories, allowing to test updates to newer SLURM packages.
+# It can also restore the latest repository.
+#
+# Variables:
+# - configure_older_updates: true to limit to older OpenHPC updates
+
+- name: Disable OpenHPC updates repository
+  ini_file:
+    dest: /etc/yum.repos.d/OpenHPC.repo
+    section: OpenHPC-updates
+    option: enabled
+    value: 0
+  when: configure_older_updates is defined and configure_older_updates|bool
+
+- name: Enable repositories providing older updates
```

Comment:
Too bad it appears we can't get hold of older versions of the openhpc-release rpm.

Reply:
We can get older ones, but their Slurm packages are completely different and not supported by our openhpc role.